controllers/feeds/commentVoteV2.js

```diff
@@ -10,10 +10,8 @@ const v = new Validator();
 async function getResult(req, reaction, dbVote, targetFeeds) {
   let result;
   const {reaction_id, vote} = req.body;
+
   if (!dbVote) {
-    if (req.userId !== reaction.user_id) {
-      targetFeeds = [...targetFeeds, `notification:${req.userId}`];
-    }
     await VoteComments.create({comment_id: reaction_id, user_id: req.userId, status: vote});
     if (vote === 'upvote') {
       const count_upvote = reaction.data.count_upvote + 1;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code creates a `VoteComments` entry and increments the `count_upvote` if the vote is an 'upvote'. However, it doesn't seem to handle the case where the vote might be a 'downvote'. If your application supports downvoting, you should also decrement the `count_upvote` when a downvote occurs. Here's a suggested fix:

```diff
  if (!dbVote) {
    await VoteComments.create({comment_id: reaction_id, user_id: req.userId, status: vote});
    let count_upvote = reaction.data.count_upvote;
+   if (vote === 'upvote') {
+     count_upvote++;
+   } else if (vote === 'downvote') {
+     count_upvote--;
+   }
  }
```

Also, note that the variable `count_upvote` is declared but not used anywhere after its declaration. You may want to update the `reaction` object or persist this updated count in your database.

Lastly, the removal of the condition `if (req.userId !== reaction.user_id)` means that users can now vote on their own comments. If this is not intended behavior, consider adding this check back in.


<!-- This is an auto-generated comment by OSS CodeRabbit -->